extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -1,10 +1,310 @@
+
+[sourcecode:html]
+<amp-story-interactive-quiz
+    id="arts-quiz"
+    endpoint="https://endpoint.com/v1/interactives"
+    prompt-text='Who was the artist that created the famous painting "The Last Supper"?'
+    option-1-text="Michelangelo"
+    option-2-text="Leonardo da Vinci" option-2-correct option-2-confetti="🎨"
+    option-3-text="Rahael"
+    option-4-text="Sandro Boticelli">
+</amp-story-interactive-quiz>
+[/sourcecode]
+
+### amp-story-interactive-results
+
+The amp-story-interactive-results element provides an interface to display a custom state depending on the options selected on the previous pages of a story.
+Requires polls or quizzes in previous pages to feed into the state of the element, and the state will be calculated according to either the quizzes (if thresholds are specified) or the polls (if only categories are specified) that the user answered.
+Each category can specify an image, title and description that will be shown when the component selects that category for the user.
+
+<amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/results-raw.png" layout="intrinsic" width="400" height="500">
+
+[sourcecode:html]
+<amp-story-interactive-results
+    theme="dark"
+    prompt-text="You are a"
+    option-1-text="Dog" option-1-image="dog.png"
+    option-1-description="You always have energy and love being with friends. Outdoors is your favorite place"
+    option-2-text="Cat" option-2-image="cat.png"
+    option-2-description="Cats are great animals for WFH">
+</amp-story-interactive-results>
+[/sourcecode]
+
+## Attributes
+
+The components have a shared API language for customizing their options. This makes it easier to port the configuration steps across all the interactive components.
+
+### id (required for binary-poll, poll, quiz)
+
+Identifies the interactive component, and is used to compose the interactiveId that is sent to the backend. Should be unique for each component in the story.
+
+### endpoint (required for binary-poll, poll, quiz)
+
+Complete URL of the backend, where votes for the interactive components are stored.
+
+### theme (optional)
+
+Controls the color of the chips and text. Can be light (default), dark.
+
+### chip-style (optional for poll, quiz, results)
+
+Controls the style of the component. Can be flat (default), shadow, or transparent.
+Results components don't support shadow.
+
+### prompt-text (optional)
+
+Adds a prompt on top of the component, useful for writing the question to a quiz or poll. Also used to write a prompt to the results component before the category title (highly encouraged).
+
+### prompt-size (optional for binary-poll, poll, quiz)
+
+Controls the font-size of the prompt text. Can be small, medium (default), large. Large prompts will hold up to 3 lines of text, other sizes will hold up to 4 lines of text.
```

Comment:
Same thing as before, I added backticks to explain that you add those values to the attribute. I don't think the pixel sizes are necessary, as we provide many other places if people want to check the sizes. What's more important is that if there's a lot of text, then creators should set the size to small, and if there's very few words, they should set it to large (so that it looks good). It's a purely creative decision that the way people will get exposure to it is through storymaking tools as a dropdown for instance. Does that make sense or should we still add the pixel values?